primer/brand
# [a11y] Tab action order for `FAQ Groups` according to specification

## Summary

This PR contains a fix update to `FAQ Groups` tab order accessibility issue 👇 . Tab list items should be navigateable by arrow keys and only one element should be focusable, based on this specification https://www.w3.org/WAI/ARIA/apg/patterns/tabs/ 

- Closes: https://github.com/github/accessibility-audits/issues/7964

I would love to ship this ASAP 🙏 



## List of notable changes:



- **added** arrow up and down navigation of vertical tabs
- **added** tabs are no `tabindex="-1"` except the active one - this fixes the a11y tab action order issue

## What should reviewers focus on?



- Review added functionality, it shoul completely align with the https://www.w3.org/WAI/ARIA/apg/patterns/tabs/

## Steps to test:



1. Navigate to `/story/components-faq-features--groups` story in the Storybook
2. Pressing the <kbd>Tab</kbd> key and <kbd>Arrow Up</kbd> and <kbd> Arrow Down </kbd>, confirm that the navigation of the tabs is according to specification https://www.w3.org/WAI/ARIA/apg/patterns/tabs/

## Supporting resources (related issues, external links, etc):

- https://github.com/github/accessibility-audits/issues/7964#issuecomment-2123542525
- https://www.w3.org/WAI/ARIA/apg/patterns/tabs/

## Contributor checklist:


## Reviewer checklist:
